azure-pipelines-product-construction-service.yml

```diff
@@ -47,10 +47,35 @@ variables:
     value: "Darc: Maestro Staging"
   - name: MaestroAppId
     value: $(MaestroStagingAppClientId)
-  - ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/main') }}:
+  - name: redisConnectionString
```

Comment:
I think this file needs a trigger for the `production` branch

Reply:
yup, added